lib/app/repo.ex

```diff
@@ -2,4 +2,39 @@ defmodule App.Repo do
   use Ecto.Repo,
     otp_app: :app,
     adapter: Ecto.Adapters.Postgres
+
+  @doc """
+  `validate_order/1` validates the ordering is one of `asc` or `desc`
+
+  ## Examples
+
+      iex> App.Repo.validate_order("asc")
+      true
+
+      iex> App.Repo.validate_order(:asc)
+      true
+
+      iex> App.Repo.validate_order(:invalid)
+      false
+
+      # Avoid common SQL injection attacks:
+      iex> App.Repo.validate_order("OR 1=1")
+      false
+  """
+  def validate_order(order) when is_bitstring(order) do
```

Comment:
From what I see, `Repo.validate_order/1` is used in `person_with_item_and_timer_count/2` and `llist_person_tags_complete/2` to validate if the sorting order is correct. 

I 100% agree that we should write this function so it can be used anywhere in the project. However, I'm wondering if it's necessary to have this function at all. 

Aren't the aforementioned functions invoked programmatically? What are the scenarios where the sorting order that is passed as arguments to these functions is other than `:asc` and `:desc`? From what I see, this is only possible if the user intentionally messes with the Javascript in the `table_component.html.heex`. If that's the case, [I'd let it crash](https://tinytechtuts.com/2021-let-it-crash-explained/).

Reply:
I think you might be up t o something here.

I agree, I'll remove this method and just leave the code as-is.

Reply:
Removed, thank you for the review!

I'll write the BUILDIT.md so this can be ready

Reply:
Thank you @panoramix360 ! Once it's ready, you can ping me and I'll approve it and get it merged :)